### PR TITLE
Upgrade python, django and other package requirements

### PR DESCRIPTION
**Pull request by mforner13**
This PR upgrades the python and django requirements to 3.9.13 and <=4.2 respectively. It also upgrades all other required packages to latest versions.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,49 +5,51 @@
...
-    long_description=open('README.md').read(),
-    long_description_content_type='text/markdown',
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
     include_package_data=True,
     install_requires=[
-        'django>=3.2.18,<4.0.0',
-        'djangorestframework>=3.4.7,<4.0.0',
-        'mohawk>=0.3.4,<2.0.0',
+        "django>=3.2.18,<=4.2",
```
Comment (timothyPatterson) [inline setup.py:19]: Should this be >=3.2.19 (since 3.2.18 has the vulnerability)?

File: setup.py [hunk 2]
```diff
@@ -5,49 +5,51 @@
...
-            'pytest==6.1.0',
-            'pytest-cov==2.10.1',
-            'pytest-django==3.10.0',
-            'flake8==3.8.3',
-            'wheel>=0.31.0,<1.0.0',
-            'setuptools>=38.6.0,<39.0.0',
-            'codecov',
-            'twine',
+        "test": [
+            "pytest==7.*",
```
Comment (timothyPatterson) [inline setup.py:25]: Should these be >=7.*? i.e. does == give flexibility?

**Review by timothyPatterson — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (mforner13) [reply setup.py:19]: Excellent point! Will update

File: setup.py (hunk 2 above)
Comment (timothyPatterson) [reply setup.py:25]: Thanks

**Pull request merged**